docs/management/manage-intro.asciidoc

```diff
@@ -15,4 +15,5 @@ include::{security-docs-root}/docs/management/admin/event-filters.asciidoc[level
 include::{security-docs-root}/docs/management/admin/host-isolation-exceptions.asciidoc[leveloffset=+1]
 include::{security-docs-root}/docs/management/admin/blocklist.asciidoc[leveloffset=+1]
 include::{security-docs-root}/docs/management/admin/endpoint-artifacts.asciidoc[leveloffset=+1]
-include::{security-docs-root}/docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc[leveloffset=+1]
\ No newline at end of file
+include::{security-docs-root}/docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc[leveloffset=+1]
+include::{security-docs-root}/docs/management/admin/endpoint-self-protection.asciidoc[leveloffset=+1]
```

Comment:
These pages feel a little out of place here. They're admin-level setup stuff, not UI-level features like the rest of this section (response console, trusted apps, event filters, etc.). Maybe "Get started > Endpoint requireements" or some other sub-section in "Get started"?

Reply:
This is a good point and I'm making a note of it, but I'd like to do it in a separate PR to keep this one focused, if that's alright.